package.json

```diff
@@ -1,97 +1,98 @@
+        "gh-pages-force": "git push origin `git subtree split --prefix dist/example`:gh-pages --force",
+        "build-static-docs": "webpack --progress --profile --colors --config webpack.build-static-docs.config.js",
+        "build-docs": "webpack --progress --profile --colors --config webpack.build-docs.config.js",
+        "serve-docs": "webpack-dev-server --progress --profile --colors --config webpack.develop.config.js"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/episodeyang/react-vis-graph-components.git"
+    },
+    "keywords": [
+        "react",
+        "component",
+        "es6",
+        "component",
+        "react-docgen",
+        "docgen",
+        "webpack",
+        "react-hot-reload"
+    ],
+    "author": "Ge Yang <yangge1987@gmail.com>",
+    "license": "ISC",
+    "bugs": {
+        "url": "https://github.com/episodeyang/react-vis-graph-components/issues"
+    },
+    "homepage": "https://github.com/episodeyang/react-vis-graph-components#readme",
+    "devDependencies": {
+        "@episodeyang/react-highlight.js": "^1.0.3",
+        "autoprefixer": "^6.3.6",
+        "babel-cli": "^6.10.1",
+        "babel-core": "^6.7.7",
+        "babel-eslint": "6.x",
+        "babel-loader": "^6.2.4",
+        "babel-plugin-transform-class-properties": "^6.10.2",
+        "babel-plugin-transform-decorators-legacy": "^1.3.4",
+        "babel-polyfill": "^6.9.1",
+        "babel-preset-es2015": "6.x",
+        "babel-preset-react": "6.x",
+        "babel-preset-stage-2": "6.x",
+        "babelify": "^7.3.0",
+        "copy-webpack-plugin": "^3.0.1",
+        "css-loader": "^0.23.1",
+        "es6-promise": "^3.2.1",
+        "eslint": "^3.3.1",
+        "eslint-plugin-react": "5.x",
+        "file-loader": "^0.8.5",
+        "html-loader": "^0.4.3",
+        "json-loader": "^0.5.4",
+        "markdown-it": "^7.0.0",
+        "markdown-it-loader": "^0.2.0",
+        "markdown-to-jsx": "^2.0.1",
+        "node-sass": "^3.7.0",
+        "opn-cli": "1.x",
+        "postcss": "^5.0.21",
+        "postcss-loader": "^0.9.1",
+        "precss": "^1.4.0",
+        "raw-loader": "^0.5.1",
+        "react-addons-test-utils": "^15.1.0",
+        "react-component-props-table": "0.0.1",
+        "react-docgen": "^2.8.2",
+        "react-docgen-loader": "^2.9.1",
+        "react-dom": "15.x",
+        "react-hot-loader": "^3.0.0-beta.2",
+        "react-markdownit": "^2.4.0",
+        "run-browser-babel": "5.x",
+        "sass-loader": "^3.2.0",
+        "scss-loader": "0.0.1",
+        "source-map-loader": "^0.1.5",
+        "style-loader": "^0.13.1",
+        "tape": "4.x",
+        "webpack": "^1.13.1",
+        "webpack-dev-server": "^1.14.1"
+    },
+    "dependencies": {
+        "bezier-js": "^2.2.1",
+        "global": "^4.3.0"
+    },
+    "peerDependencies": {
+        "react-addons-shallow-compare": "^15.3.0",
```

Comment:
Please don't set specific versions of react addons (and thus, of react as well). Take a look at how react-vis does it